packages/web3-eth-contract/src/contract.ts

```diff
@@ -408,6 +650,43 @@ export class Contract<Abi extends ContractAbi>
 		};
 	}
 
+	/**
+	 * Gets past events for this contract.
+	 *
+	 * ```ts
+	 * const events = await myContract.getPastEvents('MyEvent', {
+	 *   filter: {myIndexedParam: [20,23], myOtherIndexedParam: '0x123456789...'}, // Using an array means OR: e.g. 20 or 23
+	 *   fromBlock: 0,
+	 *   toBlock: 'latest'
```

Comment:
lets refer to our defined `BlockTags` enum in most of examples.

Reply:
Unfortunately we can't add anchor links in the code examples.